uplink/config.go

```diff
@@ -110,7 +110,7 @@ func (c Config) GetMetainfo(ctx context.Context, identity *identity.FullIdentity
 	}
 	segments := segments.NewSegmentStore(metainfo, ec, rs, c.Client.MaxInlineSize.Int(), maxEncryptedSegmentSize)
 
-	if c.RS.ErasureShareSize.Int()*c.RS.MinThreshold%c.Enc.BlockSize.Int() != 0 {
+	if c.Enc.BlockSize.Int()%c.RS.ErasureShareSize.Int()*c.RS.MinThreshold != 0 {
```

Comment:
oh wait i'm sorry, i had this backwards in the presentation yesterday. the code is right. we want the stripes to be as small as possible so audits can be as small as possible

Reply:
hmmm. ok well it still seems like _something_ is wrong here.  If the code was correct originally, then I think the error message is misleading and might need to change.  For example, before this code change, if I get this error:

```$ uplink cp go.sum sj://test3
Fatal error: Uplink configuration error: EncryptionBlockSize must be a multiple of ErasureShareSize * RS MinThreshold```

with this uplink config:

```
# size (in bytes) of encrypted blocks
enc.block-size: 8.0 KiB
# the size of each new erasure sure in bytes
rs.erasure-share-size: 1.0 KiB
# the minimum pieces required to recover a segment. k.
rs.min-threshold: 4
```

Reply:
yeah you're right, the error message is wrong

Reply:
ok let me change the error message then.

Reply:
okay i was all wrong again